yarn-project/noir-contracts/src/contracts/token_bridge_contract/src/util.nr

```diff
@@ -1,87 +1,54 @@
-use dep::std::hash::{pedersen_with_separator, sha256};
-use dep::aztec::constants_gen::{
-    GENERATOR_INDEX__SIGNATURE_PAYLOAD,
-    GENERATOR_INDEX__L1_TO_L2_MESSAGE_SECRET,
-};
+use dep::std::hash::pedersen_with_separator;
+use dep::aztec::constants_gen::GENERATOR_INDEX__L1_TO_L2_MESSAGE_SECRET;
+// docs:start:mint_public_content_hash_nr
+use dep::aztec::hash::bytes_to_sha256_in_a_field;
 
-fn compute_secret_hash(secret: Field) -> Field {
-    // TODO(#1205) This is probably not the right index to use
-    pedersen_with_separator([secret], GENERATOR_INDEX__L1_TO_L2_MESSAGE_SECRET)[0]
-}
-
-// Computes a content hash of a deposit/mint_private message.
+// Computes a content hash of a deposit/mint_public message.
 // Refer TokenPortal.sol for reference on L1.
-fn get_mint_private_content_hash(amount: Field, secret_hash_for_redeeming_minted_notes: Field, canceller: Field) -> Field {
+fn get_mint_public_content_hash(owner_address: Field, amount: Field, canceller: Field) -> Field {
```

Comment:
If having the above mentioned, most ofthis could really be discarded which would be quite nice I think.